implementations/rust/ockam/ockam_command/src/node/create.rs

```diff
@@ -105,6 +105,14 @@ pub struct CreateCommand {
 
     #[arg(long = "identity", value_name = "IDENTITY")]
     identity: Option<String>,
+
+    /// Enable AWS KMS integration.
+    #[arg(long, default_value = "false")]
+    aws_kms: bool,
+
+    /// Use an existing AWS KMS key.
+    #[arg(long)]
+    key_id: Option<String>,
```

Comment:
Just my opinion here,
these `aws_kms` and `key_id` are  used if there is no vault already,   I _feel_ it is clearer if we don't allow to automatically create kms vaults here,  and instead go only with the simplest one (the file based one,  on the default location).     To use a kms vault as the default one for a node,   first explicitly create the vault.    For example we allow to use a different file system path than the default for storage,  but don't have that option here on node create  (so to use a different path,  explicitly create the vault first).

Reply:
I agree on this. We want to provide "defaults" as much as possible, but only to cover the simplest/default use case, and the KMS feature is not part of that.